superflore/docker.py

```diff
@@ -36,9 +37,19 @@ def map_directory(self, host, container=None, mode='rw'):
     def add_bash_command(self, cmd):
         self.bash_cmds.append(cmd)
 
+    def clear_commands(self):
+        self.bash_cmds = list()
+
     def build(self):
+        if not self.dockerfile_directory:
```

Comment:
This looks like the only place that the dockerfile is used so maybe it should be an argument for this method instead of a constructor argument?

Reply:
Yep, that's a good point.